client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -16,12 +36,17 @@ const Sidebar = ( { siteSlug }: { siteSlug: string | null } ) => {
 				<span className="launchpad__sidebar-header-flow-name">Newsletter</span>
 			</div>
 			<div className="launchpad__sidebar-content-container">
-				<div className="launchpad__progress-bar-container">
-					<span className="launchpad__progress-value">33%</span>
-					<div className="launchpad__progress-bar">
-						<div className="launchpad__progress-bar-completed" />
+				{ taskCompletionProgress && (
+					<div className="launchpad__progress-bar-container">
+						<span className="launchpad__progress-value">{ taskCompletionProgress }%</span>
+						<ProgressBar
```

Comment:
Is it better to not show the progress bar when taskCompletionProgress comes back null? Or better to show it at 0%? 🤔  I cant decide my opinion either way, but this seems good!

Reply:
I think its better to hide it right now to match the checklist logic. If the task list is empty / non-existant the both the checklist and progress bar will be hidden.